content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -7,7 +7,7 @@ dbSwitcher: ['postgresql', 'mysql', 'sqlserver', 'planetscale', 'cockroachdb']
 toc: false
 ---
 
-## Introspect your database with Prisma
+## Introspect your database with Prisma ORM
```

Comment:
@grpatel can we check for broken links referencing this?

Reply:
Hopefully, there aren't that many broken links. If you want to do a pass for broken links. You can make a PR on this branch that will merge into it when we merge this one. If that makes it easier for you. @grpatel